source/index.html

```diff
@@ -1,39 +1,282 @@
+                    <th colspan="4">Daycare</th>
+                    <tr>
+                        <td>M</td>
+                        <td>7-7</td>
+                        <td>Th</td>
+                        <td>7-7<span>*</span></td>
+                    </tr>
+                    <tr>
+                        <td>T</td>
+                        <td>7-7</td>
+                        <td>F</td>
+                        <td>7-7<span>*</span></td>
+                    </tr>
+                    <tr>
+                        <td>W</td>
+                        <td>7-7</td>
+                        <td>S</td>
+                        <td>7-7*</td>
+                    </tr>
+                    <td colspan="4">*by appointment</td>
+                </table>
+                <div class="contact-us">
+                    <a href="https://www.google.com/maps/place/Poets+Pet+Parlor/@46.5646401,-87.415697,17z/data=!3m1!4b1!4m5!3m4!1s0x4d51df4d52a6b0cd:0xfc299af6713a58b7!8m2!3d46.5646401!4d-87.4135083"><img src="./assets/location.svg"></a>
+                    <a href="tel:(906)228-2708"><img src="./assets/phone.svg"></a>
+                    <a href="mailto:poetspetparlor@gmail.com"><img src="./assets/email.svg"></a>
+                </div>
+                <p class="philosophy">Poets pet parlor provides the only 24 hour staffed care for your pet in Marquette County. Offering full service grooming, doggie daycare and luxury boarding, in a
+                safe, fun, and homelike environment. At poets pet parlor, we treat your pet like our own.</p>
+            </div>
+        </header>
+        <section id="about-me">
+            <img class="dog-divider" src="./assets/new-dog1.png">
+            <hr>
+            <div class="content">
+                <h2>ABOUT ME &amp POET</h2>
+                <p>I am the proud owner of 4 mix breed dogs, Oli, Saige, Lucy and Sunny, two macaws, and four cats whom I love with all her heart. They are my family, and companions, and my inspiration for Starting Poets Pet Parlor. I began as a dog groomer who wanted to offer cage free options for grooming and as my clientele grew, I was asked to watch and care for many of my clients pets who did not want to leave their pets at a traditional kennel. I expanded my business to include doggie daycare and now pet boarding that is aimed to provide the most homelike environment for your dog as possible. As an alternative to the traditional dog kennel, I have created a safe and fun environment where your dog can romp with friends in indoor and outdoor play yards, sleep in a home-like bedroom, or even sleep in an on-site apartment in bed with staff.</p>
+                <p>Poet was my first dog and namesake for Poets Pet Parlor. He was an amazing dog who began my love for animals. He passed away 2 months after the opening of Poets.</p>
+            </div>
+        </section>
+        <section id="clients">
+            <img class="dog-divider" src="./assets/new-dog5.png">
```

Comment:
I'm not sure what they should be, but if you have names/breed/short descriptions for the dogs it'd be really great to put that into the `alt` attribute. I don't know, probably not a big deal either.